backend/tracim_backend/command/cleanup.py

```diff
@@ -0,0 +1,248 @@
+            deleted_user_id = cleanup_lib.anonymise_user(
+                user, anonymised_user_display_name=anonymised_user_display_name
+            ).user_id
+            print(
+                'user {} anonymised to "{} <{}>".'.format(
+                    user.user_id, user.display_name, user.email
+                )
+            )
+        elif not_owned_workspace_revisions and not best_effort and not force:
+            print(
+                "ERROR: User {} has revisions in other user workspace. Cannot delete it without"
+                " creating inconsistent database. Rollback changes.".format(user.user_id)
+            )
+            exit(3)
+        else:
+            print('Delete all user "{}" data in database'.format(user.user_id))
+            # INFO - G.M - 2019-12-13 - We can delete full user data
+            deleted_workspace_ids = cleanup_lib.delete_user_owned_workspace(user)
+            deleted_user_id = cleanup_lib.delete_full_user(user)
+        self._session.flush()
+        return DeleteResultIds(deleted_user_id, deleted_workspace_ids)
+
+
+class AnonymiseUserCommand(AppContextCommand):
+    def get_description(self) -> str:
+        return """anonymise user from database"""
+
+    def get_parser(self, prog_name: str) -> argparse.ArgumentParser:
+        parser = super().get_parser(prog_name)
+        parser.add_argument(
+            "--dry-run",
+            help="dry-run mode",
+            dest="dry_run_mode",
+            default=False,
+            action="store_true",
+        )
+        parser.add_argument(
+            "--anonymous-name",
+            help="Anonymous user display name to use",
+            dest="anonymous_name",
+            required=False,
+        )
+        parser.add_argument(
+            "-l", "--login", nargs="+", help="User logins (email)", dest="logins", required=True
+        )
+        return parser
+
+    def take_app_action(self, parsed_args: argparse.Namespace, app_context: AppEnvironment) -> None:
+        self._session = app_context["request"].dbsession
+        self._app_config = app_context["registry"].settings["CFG"]
+
+        if parsed_args.dry_run_mode:
+            print("(!) Running in dry-run mode, not change will be applied.")
+
+        with unprotected_content_revision(self._session) as session:
+            uapi = UserApi(
+                config=self._app_config,
+                session=session,
+                current_user=None,
+                show_deleted=True,
+                show_deactivated=True,
+            )
+            user_list = []  # type: typing.List[User]
+            for login in parsed_args.logins:
+                try:
+                    user = uapi.get_one_by_email(login)
+                    user_list.append(user)
+                except UserDoesNotExist:
+                    print("ERROR: user with email {} does not exist".format(login))
+                    exit(2)
```

Comment:
`2` ? if use other than `1` e should document these error codes

Reply:
fixed.